tap_google_analytics/ga_client.py

```diff
@@ -240,23 +255,25 @@ def query_api(self, report_definition, pageToken=None):
         return self.analytics.reports().batchGet(
             body={
                 'reportRequests': [
-                {
-                    'viewId': self.view_id,
-                    'dateRanges': [{'startDate': self.start_date, 'endDate': self.end_date}],
-                    'pageSize': '1000',
-                    'pageToken': pageToken,
-                    'metrics': report_definition['metrics'],
-                    'dimensions': report_definition['dimensions'],
-                }]
+                    {
+                        'viewId': self.view_id,
+                        'dateRanges': [{'startDate': start_date, 'endDate': end_date}],
```

Comment:
just confirming, but in case start_date and end date are none, the API is ok with this?